src/tpm.rs

```diff
@@ -83,6 +83,15 @@ pub(crate) fn get_tpm2_ctx() -> Result<Context> {
     Context::new(tcti).map_err(|e| e.into())
 }
 
+// Holds the output of create_ek
+#[derive(Clone, Debug)]
+pub struct EKResult {
+    pub key_handle: KeyHandle,
+    pub ek_cert: Option<Vec<u8>>,
+    pub public: tss_esapi::structures::Public,
+    pub tpm2b_pub: Vec<u8>,
```

Comment:
Do we need this member (`tpm2b_pub`), as it could be calculated from `public` on demand?

Reply:
Yes. I even think that we can remove the pub_to_vec function because newer tss versions have a marschall function. See: https://docs.rs/tss-esapi/7.0.1/tss_esapi/structures/enum.Public.html#method.marshall

Reply:
Done.